docs/source/ops/connect/helm.rst

```diff
@@ -0,0 +1,281 @@
+Helm chart, waiting for all of the pods to stop, and then installing a higher
+version. Destroying all of the components is a safe operation because all of
+the state is stored in the provided database coordinates. There is no
+additional state within the components themselves.
+
+The components are not designed for running concurrently with older versions,
+so it is imperative to wait for the existing Helm chart components to be
+completely shut down before installing the new one. Do not try to upgrade in
+place.
+
+Assuming you do not change the database coordinates, you should have data
+continuity through the upgrade.
+
+Backing Up
+----------
+
+.. note::
+
+   This section only makes sense with the ``production`` flag set to ``true``.
+
+For a production setup, you should be providing the Helm chart with external
+database coordinates. The simplest approach here is to periodically back up
+those databases as a whole, just like you would any other database.
+
+If you want to be more fine-grained, you *may* decide to **not** backup the
+database used by the HTTP JSON API Service instances. Note that it is
+imperative that you still backup the databases for the other components (Trigger
+Service and Daml Driver for PostgreSQL) if you are running them.
+
+If you are running the Helm chart solely for the HTTP JSON API Service
+(connected to an external Ledger API server), then you can eschew backing up
+entirely, as the database for the HTTP JSON API Service is an
+easy-to-reconstruct cache. This assume that, in this setup, the data store of
+the Ledger API server is, itself, properly backed up.
+
+Securing Daml Connect
+---------------------
+
+The Helm chart assumes that the Kubernetes environment itself is trusted, and
+as such does not encrypt connections between components. Full TLS encryption
+between every component is not supported by the Helm chart. Individual
+components do support it, so if that is a requirement for you you can still set
+it up, though not through the Helm chart. Refer to the
+`Secure Daml Infrastructure <https://github.com/digital-asset/ex-secure-daml-infra>`_
+repository for guidance on how to set that up.
+
+When using the Helm chart, we recommend against exposing the Ledger API gRPC
+endpoint outside of the cluster, and exposing the HTTP JSON API Service,
+Trigger Service, and OAuth2 Middleware endpoints only through an HTTP proxy.
+That is why the services started by the Helm chart are of type ``ClusterIP``.
+
+That proxy should either do TLS termination, or be itself behind a proxy that
+does, in which case all of the communications between the TLS termination
+endpoint must be happening on a fully trusted network.
+
+See the :ref:`auth0` section for an example of setting up nginx to proxy
+external connections to the JSON API, Trigger Service and OAuth2 Middleware.
+
+Helm Chart Options Reference
+----------------------------
+
+TODO: copy over Configuration.md
```

Comment:
In this PR?

Reply:
Wondering too

Reply:
Yes, but I was out of fuel for a md -> rst translation yesterday.